datamade/how-to
# Add etl/ directory with documentation for marshmallow

## Overview

See title.

Handles #63

## Testing Instructions

* View the rendered markdown and ensure that it looks good.
* Click the links and confirm they resolve.
